### PR TITLE
Update to KoreBuild-dotnet

### PR DESCRIPTION
**Pull request by bricelam**
Blocked by dotnet/cli#710

@victorhurdugaci

### CODE REVIEW

**Comment by victorhurdugaci** [comment]
Niiiice!

**Comment by victorhurdugaci** [comment]
Is it blocked?

**Comment by bricelam** [comment]
Yeah, `.testPublish` doesn't contain `sqlite3.dll` so net451 tests fail.

**Comment by victorhurdugaci** [comment]
Added the issue to my tracking list here: https://github.com/aspnet/Universe/issues/324

File: test/Microsoft.Data.Sqlite.Tests/project.json [hunk 1]
```diff
@@ -6,17 +6,22 @@
...
+    "dnx451": {
+      "dependencies": {
+        "xunit.runner.aspnet": "2.0.0-aspnet-*"
+      }
+    },
     "dnxcore50": {
-      "System.IO.FileSystem": " 4.0.0-*",
-      "System.Threading.Thread": "4.0.0-*"
+      "dependencies": {
+        "xunit.runner.console": "2.1.0"
```
Comment (bricelam) [inline test/Microsoft.Data.Sqlite.Tests/project.json]: Can remove now. KoreBuild downloads it.

Comment (victorhurdugaci) [inline test/Microsoft.Data.Sqlite.Tests/project.json]: Don't assume that. It's just a hack that KoreBuild downloads it

**Comment by victorhurdugaci** [comment]
@bricelam can you workaround this based on the suggestions [here](https://github.com/dotnet/cli/issues/710)?

**Pull request merged**